test/e2e/plugins/index.js

```diff
@@ -0,0 +1,19 @@
+const promisify = require( 'util.promisify' );
```

Comment:
Fun story, Cypress server quietly uses node 6.5.0 internally. I spent a lot of time tearing my hair out, wondering why `require( 'util' ).promisify` wasn't working.